impeller/renderer/backend/gles/reactor_gles.h

```diff
@@ -260,9 +260,9 @@ class ReactorGLES {
 
   std::unique_ptr<ProcTableGLES> proc_table_;
 
-  Mutex ops_execution_mutex_;
   mutable Mutex ops_mutex_;
-  std::vector<Operation> ops_ IPLR_GUARDED_BY(ops_mutex_);
+  std::map<std::thread::id, std::vector<Operation>> ops_ IPLR_GUARDED_BY(
```

Comment:
Do we want to instead key this with `EGLContext`?  It sounded like Chinmay and Jonah had use cases in mind that revolved around passing the context between multiple threads in a worker pool?

Reply:
Currently EGL contexts are not passed between threads.

ReactorGLES does support calling `AddOperation` on a thread that does not have an active `EGLContext`.  Using the thread ID as a key allows queueing of operations when the `EGLContext` is unavailable.